cluster/gce/gci/configure-helper.sh

```diff
@@ -1288,7 +1288,7 @@ function start-kube-apiserver {
   fi
 
 
-  local authorization_mode="RBAC"
+  local authorization_mode="Node,RBAC"
```

Comment:
gce/cos is a symlink to gce/gci. I'm not sure if gce/container-linux is tested anywhere. A quick search of testgrid didn't turn anything up. I'm not inclined to change anywhere that is not being tested.